### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and settings_Keyboard shortcuts_Screenreader: Key binding status message "1 existing command has this key bending" is not announced by screen reader

### ISSUE THREAD

**Issue by v-inanda** [microsoft/vscode#159432]
### **GitHub Tags:**
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG4.1.3; #NVDA; #ScreenReader; #Narrator;

### **Environment and OS details:**
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

### **Repro Steps:**

1. Turn on NVDA
2. Open Visual studio code editor.
3. Tab till setting icon and Select "Keyboard shortcuts options"
4. Verify that Key bending status message "1 existing command has this key bending" is announced by screen reader or not.

### **Actual Result:**
Key bending status message "1 existing command has this key bending" is not announced by NVDA/Narrator screen readers.

### **Expected Result:**
NVDA/Narrator should announce the status message "1 existing command has this key bending" 

### **Attachment:**
![NVDA](https://user-images.githubusercontent.com/91525824/187140805-1d1a28be-917a-4114-989c-533cd161fb10.gif)

![narrator](https://user-images.githubusercontent.com/91525824/187140950-65067465-9833-465a-bcee-663ea4a55ce8.gif)

**Comment by isidorn** [comment]
@sandy081 can we just do an aria alert when there are existing keybindings?
This issue we have to fix since it is severity 2, though it sounds law hanging.

**Comment by v-inanda** [comment]
@sandy081 PO should not close the bug. Only trusted tester closes the bug. 
This bug is not fixed. So, please provide the reason for closing the bug. Thanks

**Comment by isidorn** [comment]
@v-inanda make sure to use VS Code insiders https://code.visualstudio.com/insiders/

**Comment by v-inanda** [comment]
Issue is fixed. So,Closing the bug.
![Screen-Recording-3](https://user-images.githubusercontent.com/91525824/189098844-c2baf5bd-9793-436a-854d-bf6e2799a22c.gif)
